Assets/MRTK/SDK/Features/UX/Scripts/Collections/GridObjectCollectionSizeFitter.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    [ExecuteInEditMode]
+    public class GridObjectCollectionSizeFitter : MonoBehaviour
+    {
+        [SerializeField]
+        [Tooltip("The GridObjectColletion to fit")]
+        private GridObjectCollection GridObjectCollection;
```

Comment:
This should fix the build issue:

```suggestion
        private GridObjectCollection GridObjectCollection = null;
```